packages/nimble-components/src/table-column/base/models/column-validator.ts

```diff
@@ -1,17 +1,17 @@
+import { observable } from '@microsoft/fast-element';
 import { Validator } from '../../../utilities/models/validator';
 import type { TableColumnValidity } from '../types';
-import type { ColumnInternals } from './column-internals';
 
 /**
  * Base column validator
  */
 export class ColumnValidator<
     ValidityFlagNames extends readonly string[]
 > extends Validator<ValidityFlagNames> {
-    public constructor(
-        private readonly columnInternals: ColumnInternals<unknown>,
-        configValidityKeys: ValidityFlagNames
-    ) {
+    @observable
+    public isColumnValid = true;
```

Comment:
I needed to add an `@observable` property here to track whether or not the column is valid because the base `Validator` does not have an observable way to know when its validity has changed, which is required by the table. Therefore, I've added `isColumnValid` here, and I mirror that in the existing `validConfiguration` property on the `ColumnInternals` object. I did this because I didn't want the table to have to listen to changes to the column, column internals, and column validator. The table is already handles changes coming from `columnInternals.validConfiguration`.